javascript/javascript1/week3/item-remove.js

```diff
@@ -0,0 +1,15 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  names.splice(1,1);  //splice method's arguments represent index and number of items to be removed
```

Comment:
I'm pretty sure a part of the assignment is to get the index of the name with an array method.
Eg. `const nameIndex = names.indexOf(nameToRemove)`

Reply:
Ohhh yes. Thanks